docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,59 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and ``source``ing the environment, there are couple of
+steps to do to run QtCreator in the SDK environment. This roadmap assumes
+that SDK and QtCreator are installed and the environment is ``source``d already.
+
+Adding a new kit
+--------------------
+
+To be able to cross-compile via QtCreator, a new build kit should be added to
+QtCreator.
+
+* Open QtCreator
+* Open Options form from Tools->Options, then click ``Build & Run`` in the left
+panel of the option window.
+* Select the ``Kits`` tab and click the ``Add`` button.
+* Name your new kit and as ``Device type`` choose ``GenericLinuxOsType``.
+* Then browse and select the path to the sysroot. If you don't know the sysroot
+location you can discover by typing ``echo ${SDKTARGETSYSROOT}`` to the terminal.
+Please note that the terminal should be ``source``d beforehand.
+* Then choose C compiler. If the cross compiler of the SDK is not listed, it should
+be added as a new gcc. To add new gcc click the ``Manage`` button next to compiler.
+Add your GCC path here. You can discover this path from environment as well.
+To do so, type ``which ${CC}`` in your terminal. Please note that the terminal
+should be ``source``d beforehand.
+* Choose the C++ compiler as you did in the previous step. You can discover path by
+typing ``which ${CXX}`` to the terminal. Please note that the terminal should be
+``source``d beforehand.
+* Copy the environment list of the SDK to ``Environment``. You can find the environment
+list by typing ``env`` to the terminal.
+* Choose ``Debugger``, if it is not listed in the dropdown menu click the ``Manage..``
+and add a new Debugger. You can find the debugger path with command ``which $GDB``.
+Please note that the terminal should be ``source``d beforehand.
+* Choose CMake Tool, if it is not listed in the dropdown menu click the ``Manage..``
+button and add the path to CMake. You can find the path to Cmake with ``which cmake``.
+Please note that the terminal should be ``source``d beforehand.
+* If you are working with qmake then it should be added as well. You can discover
+the path to qmake with ``which qmake``. Please note that the terminal should be
+``source``d beforehand.
+* Add ``CMAKE_SYSROOT:STRING=<path of sysroot>`` item to the ``CMake Configuration``
```

Comment:
"Add the ..."
\<path of sysroot\> should be \<sysroot path\> or \<path to sysroot\>